recipe/comparison.patch

```diff
@@ -9,7 +9,7 @@
 +        METRIC=$(compare -metric DSSIM images/$png bad/$png null 2>&1);
  
 -        if [[ $compare_ret -eq 0 ]]; then
-+        if [[ $(echo "$METRIC < 10^-5" | sed 's/e-0*/*10^-/' |bc -l) -eq 1 ]]; then
++        if [[ $(echo "$METRIC < 10^-5" | sed -E 's/.*\((.+)\)/\1/' | sed 's/e-0*/*10^-/' |bc -l) -eq 1 ]]; then
```

Comment:
Output format of `compare -metric DSSIM` has changed from `9.8793e-07` to `0.064744 (9.8793e-07)` on recent versions of imagemagick